engine/src/test/java/org/apach/apex/stram/DeployRequestTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apach.apex.stram;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.apache.apex.stram.DeployRequest;
+
+public class DeployRequestTest
+{
+  private DeployRequest underTest;
+
+  @Before
+  public void setup()
+  {
+    underTest = new DeployRequest("000001");
+  }
+
+  @Test
+  public void testAddAffectedContainer()
```

Comment:
Not very useful to create unit tests for trivial code like here. Unit tests should test non-trivial logic in a unit and if that logic includes method such as addAffectedContainers(...) they automatically get tested.